test/parallel/test-buffer.js

```diff
@@ -749,6 +749,11 @@ for (let i = 0; i < 256; i++) {
   assert.equal(hexb2[i], hexb[i]);
 }
 
+//#6770 Test single hex character throws TypeError
```

Comment:
You can reference github issues in comments, but it is usually preferred to give the full URL for things like that so that it’s easier to look up and there is no ambiguity wrt the old issue tracker at https://github.com/nodejs/node-v0.x-archive/